custom/aaa/tests/data/agg_tables/woman.csv

```diff
@@ -3,5 +3,5 @@ reach-test,state_1,district_1,block_1,supervisor_1,awc_1,taluka_1,phc_1,sc_1,vil
 reach-test,state_1,district_1,block_1,supervisor_1,awc_1,taluka_1,phc_1,sc_1,village_1,household_case_1,non_institutional_delivery,2018-07-01,,"[DateRange(datetime.date(2018, 7, 14), datetime.date(2019, 1, 24), '[)')]",Non Institutional Delivery,2000-01-01,married,F,,,,False,husband guy,,,,super cool house,hindu,sc,apl
 reach-test,state_1,district_1,block_1,supervisor_1,awc_1,taluka_1,phc_1,sc_1,village_1,household_case_1,pregnant_woman,2018-08-01,,"[DateRange(datetime.date(2018, 8, 1), None, '[)')]",Pregnant Lady,2000-01-01,married,F,,,,False,,,,,super cool house,hindu,sc,apl
 reach-test,state_1,district_1,block_1,supervisor_1,awc_1,taluka_1,phc_1,sc_2,village_2,household_case_3,high_risk_pregnant_woman,2017-07-01,,"[DateRange(datetime.date(2018, 9, 14), None, '[)')]",High Risk Pregnant Woman,1992-07-04,,F,,,,True,,,,,,,,
-reach-test,state_1,district_1,block_2,supervisor_2,awc_2,taluka_1,phc_1,sc_1,village_1,household_case_2,ec_using_condoms,2019-01-01,,,Eligible Couple using Condoms,2000-01-01,married,F,,"[DateRange(datetime.date(2019, 2, 3), None, '[)')]",,False,,8675309,1,,,,,
+reach-test,state_1,district_1,block_2,supervisor_2,awc_2,taluka_1,phc_1,sc_1,village_1,household_case_2,ec_using_condoms,2019-01-01,,,Eligible Couple using Condoms,2000-01-01,married,F,,"[DateRange(datetime.date(2019, 2, 3), datetime.date(2019, 2, 4), '[)'), DateRange(datetime.date(2019, 2, 4), None, '[)')]",,False,,8675309,1,,,,,
```

Comment:
This DateRange text means:

two date ranges

1. 2019-02-03 - 2019-02-04 [inclusive, exclusive)
2. 2019-02-04 - (no end) [inclusive, exclusive)